snuba/cli/health.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import annotations
+
+import itertools
+import logging
+import sys
+import time
+from collections import defaultdict
+from typing import Any, List, Mapping, MutableMapping, Set, Union, cast
+
+import click
+import simplejson as json
+
+from snuba import settings
+from snuba.clickhouse.errors import ClickhouseError
+from snuba.clusters.cluster import (
+    ClickhouseClientSettings,
+    ConnectionId,
+    UndefinedClickhouseCluster,
+)
+from snuba.datasets.factory import get_dataset, get_enabled_dataset_names
+from snuba.datasets.schemas.tables import TableSchema
+from snuba.datasets.storage import Storage
+from snuba.environment import setup_logging
+from snuba.web.views import check_down_file_exists, metrics
+
+
+@click.command()
```

Comment:
instead of removing the endpoint, the api and this command should share code. Pull out the healthcheck code into its own utility function and have both the cli and endpoint call that utility function

Reply:
The bulk of the health checking logic is now in `health_info.py`'s `get_health_info` function. The cli and the endpoint functions calls `get_health_info`.